### ISSUE TITLE
Calculate miou on training data set

### ISSUE THREAD

**Issue by SharhadBashar** [zengxianyu/jsws#15]
I noticed that miou is only calculated on validation data set
What would i have to do if I want to calculate the miou on the training data set as well?

**Comment by zengxianyu** [comment]
you could change the path from val set to training set

**Comment by SharhadBashar** [comment]
Thats exactly what i did
Shouldn't the training set miou be better than validation set miou?
My results show training set is the same as validation, and in some cases its worse
